clusterloader2/pkg/measurement/measurement_executor.go

```diff
@@ -24,18 +24,26 @@ import (
 )
 
 // Execute executes a measurement, which can be a single measurement or a wrapper for multiple measurements.
-// It throws an error if both identifier and Instances has been supplied in the config.
-func Execute(mm Manager, m *api.Measurement) error {
-	if len(m.Instances) != 0 && m.Identifier != "" {
-		return fmt.Errorf("only one of instances or identifier must be supplied. Measurement method - %s, identifier - %s, instances - %v", m.Identifier, m.Method, m.Instances)
-	}
+func Execute(mm Manager, m *api.Measurement) errors.ErrorList {
 	if m.Identifier != "" {
-		return mm.Execute(m.Method, m.Identifier, m.Params)
+		return executeSingleMeasurement(mm, m)
 	}
 	return executeWrapperMeasurement(mm, m)
 }
 
-func executeWrapperMeasurement(mm Manager, m *api.Measurement) error {
+func formatError(method, identifier string, err error) error {
```

Comment:
Are we going to use it anywhere else? Right now it is only used once as far as I can see.

Reply:
Done, but I think we should refactor this code and test it better.